lib/chef/shell.rb

```diff
@@ -108,9 +108,9 @@ def self.configure_irb
     irb_conf[:IRB_RC] = lambda do |conf|
       m = conf.main
 
-      conf.prompt_c       = "chef#{leader(m)} > "
+      conf.prompt_c       = "chef#{leader(m)} (#{Chef::VERSION})> "
```

Comment:
Looking at http://ruby-doc.org/stdlib-2.0/libdoc/irb/rdoc/IRB.html, it seems like prompt_c "prompt for continuated statement" would be excessive?